microsoft-edge/devtools-guide-chromium/shortcuts/index.md

```diff
@@ -75,96 +75,96 @@ The following keyboard shortcuts are available in most DevTools panels.
-| Turn on or off the next occurrence of whatever word the cursor is on.  Each occurrence is highlighted simultaneously | `Ctrl`+`D` / `Ctrl`+`U` | `Command`+`D` / `Command`+`U` |
+| Delete all characters in the last word, up to the cursor | **Ctrl+Delete** | **Option+Delete** |
+| Add or remove a [line-of-code breakpoint](../javascript/breakpoints.md#line-of-code-breakpoints) | Focus your cursor on the line, then press **Ctrl+B** | Focus your cursor on the line, then press **Command+B** |
+| Navigate to matching bracket | **Ctrl+M** | **Ctrl+M** |
+| Toggle single-line comment.  If multiple lines are selected, DevTools add a comment to the start of each line | **Ctrl+/** | **Command+/** |
+| Turn on or off the next occurrence of whatever word the cursor is on.  Each occurrence is highlighted simultaneously | **Ctrl+D** / **Ctrl+U** | **Command+D** / **Command+U** |
 
 
 <!-- ====================================================================== -->
 ## Performance tool keyboard shortcuts
 
 | Action | Windows/Linux | macOS |
 |---|---|---|
-| Start / stop recording | `Ctrl`+`E` | `Command`+`E` |
-| Save recording | `Ctrl`+`S` | `Command`+`S` |
-| Load recording | `Ctrl`+`O` | `Command`+`O` |
+| Start / stop recording | **Ctrl+E** | **Command+E** |
+| Save recording | **Ctrl+S** | **Command+S** |
+| Load recording | **Ctrl+O** | **Command+O** |
 
 
 <!-- ====================================================================== -->
 ## Memory tool keyboard shortcuts
 
 | Action | Windows/Linux | macOS |
 |---|---|---|
-| Start / stop recording | `Ctrl`+`E` | `Command`+`E` |
+| Start / stop recording | **Ctrl+E** | **Command+E** |
 
 
 <!-- ====================================================================== -->
 ## Console tool keyboard shortcuts
 
 | Action | Windows/Linux | macOS |
 |---|---|---|
-| Accept autocomplete suggestion | `Right Arrow` or `Tab` | `Right Arrow` or `Tab` |
-| Reject autocomplete suggestion | `Escape` | `Escape` |
-| Get previous statement | `Up Arrow` | `Up Arrow` |
-| Get next statement | `Down Arrow` | `Down Arrow` |
-| Focus the **Console** | `Ctrl`+ `` ` `` | `Ctrl`+`` ` `` |
-| Clear the **Console** | `Ctrl`+`L` | `Command`+`K` or `Option`+`L` |
-| Force a multi-line entry.  This shortcut is mostly unnecessary, because DevTools should detect multi-line scenarios by default | `Shift`+`Enter` | `Command`+`Return` |
-| Run | `Enter` | `Return` |
-| Expand all subproperties of an object that are logged to the Console | Hold `Alt`, then click **Expand** (![Expand](../media/expand-icon.msft.png)) | Hold `Alt`, then click **Expand** (![Expand](../media/expand-icon.msft.png)). |
+| Accept autocomplete suggestion | **Right Arrow** or **Tab** | **Right Arrow** or **Tab** |
+| Reject autocomplete suggestion | **Esc** | **Esc** |
+| Get previous statement | **Up Arrow** | **Up Arrow** |
+| Get next statement | **Down Arrow** | **Down Arrow** |
+| Focus the **Console** | **Ctrl+`** | **Ctrl+`** |
```

Comment:
The ` character breaks the rendering here. Screenshot of the problem:

<img width="485" alt="image" src="https://user-images.githubusercontent.com/1152698/227984698-ed0b0756-2a84-4a02-9995-2484a5b569ae.png">

You need to escape the two backticks on this line to avoid markdown thinking that what's in between them is code.
This should work:

```suggestion
| Focus the **Console** | **Ctrl+\`** | **Ctrl+\`** |
```

Reply:
FTS 
![image](https://user-images.githubusercontent.com/45407486/228023297-f1f12a6c-1d57-469e-86ee-392f2bc1a98f.png)